api_generator/src/bin/run.rs

```diff
@@ -28,9 +28,9 @@ use std::{
 };
 
 fn main() -> Result<(), failure::Error> {
-    // This must be run from the src root directory, with cargo run -p api_generator
+    // This must be run from the repo root directory, with cargo run -p api_generator
```

Comment:
Guess we can update this now to `cargo make generate-api` 😃

Reply:
Indeed 😉 - done in 00efb77